pkg/util/ovs/ovs.go

```diff
@@ -199,7 +199,7 @@ func (ovsif *ovsExec) AddPort(port string, ofportRequest int, properties ...stri
 	}
 	ofport, err := ovsif.GetOFPort(port)
 	if ofportRequest > 0 && ofportRequest != ofport {
-		return -1, fmt.Errorf("Allocated ofport (%d) did not match request (%d)", ofport, ofportRequest)
+		return -1, fmt.Errorf("Allocated ofport (%d) did not match request (%d): %v", ofport, ofportRequest, err)
```

Comment:
we should probably have a separate message for the err!=nil case; this error message was basically written under the assumption that GetOFPort will always succeed